### PR TITLE
Update mage.go commands in README.md

### PR DESCRIPTION
**Pull request by andremralves**

### CODE REVIEW

**Review by jcchavezs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,8 +15,9 @@ OWASP Coraza WAF is 100% compatible with OWASP Coreruleset and Modsecurity synta
 ```bash
 ▶ go run mage.go -l
 Targets:
-  build              builds the plugin.
-  buildLinux         builds the plugin with GOOS=linux.
+  buildCaddy         Build builds the plugin.
```
Comment (M4tteoP) [inline README.md:18]:
```suggestion
  buildCaddy        builds the plugin.
```

File: README.md [hunk 2]
```diff
@@ -15,8 +15,9 @@ OWASP Coraza WAF is 100% compatible with OWASP Coreruleset and Modsecurity synta
...
+  buildCaddyLinux    BuildLinux builds the plugin with GOOS=linux.
```
Comment (M4tteoP) [inline README.md:19]:
```suggestion
  buildCaddyLinux   builds the plugin with GOOS=linux.
```

**Review by M4tteoP — COMMENTED** [review]
Thanks for this PR! I think these lines should be printed as the suggested changes and the following comment lines updated accordingly:
- https://github.com/corazawaf/coraza-caddy/blob/main/magefile.go#L135 (`// BuildCaddy builds the plugin.`)
- https://github.com/corazawaf/coraza-caddy/blob/main/magefile.go#L140 (`// BuildCaddyLinux builds the plugin.`)

Would you mind updating also them?

Many thanks!

**Comment by andremralves** [comment]
Hello @M4tteoP, thanks for the review. I made the changes and also updated the descriptions in `magefile.go`!

**Comment by M4tteoP** [comment]
Thanks @andremralves!

**Pull request merged**